tests/manage/noobaa/conftest.py

```diff
@@ -0,0 +1,111 @@
+import logging
+
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources import noobaa
+from tests import helpers
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def noobaa_obj():
+    """
+    Returns a NooBaa resource that's connected to the S3 endpoint
+    Returns:
+        NooBaa: A NooBaa resource
+
+    """
+
+    noobaa_obj = noobaa.NooBaa()
+    return noobaa_obj
+
+
+@pytest.fixture()
+def uploaded_objects(request, noobaa_obj, awscli_pod):
+    """
+    Deletes all objects that were created as part of the test
+
+    Returns:
+        list: An empty list of objects
+
+    """
+
+    uploaded_objects = []
+
+    base_command = (
+        f"sh -c \"AWS_ACCESS_KEY_ID={noobaa_obj.access_key_id} "
+        f"AWS_SECRET_ACCESS_KEY={noobaa_obj.access_key} "
+        f"AWS_DEFAULT_REGION={noobaa_obj.region} "
+        f"aws s3 "
+        f"--endpoint={noobaa_obj.endpoint} "
+    )
+
+    def object_cleanup():
+        for uploaded_filename in uploaded_objects:
+            # TODO: Add assert
+            logger.info(f'Deleting object {uploaded_filename}')
+            awscli_pod.exec_cmd_on_pod(
+                command=base_command + "rm " + uploaded_filename + "\"",
+                secrets=[noobaa_obj.access_key_id, noobaa_obj.access_key, noobaa_obj.endpoint]
+            )
+    request.addfinalizer(object_cleanup)
+    return uploaded_objects
```

Comment:
that's confusing,
the fixture name and the function attribute have the same name so not clear what is returned here